docs/operation.md

```diff
@@ -217,6 +217,12 @@ The combined view displays suppressed bins. The `Count (anonymized)` column is l
 
 The combined view lets you examine precisely the distortion and suppression. The anonymized data can be sorted by each column. Sorting by `Count (original)` shows the ascending suppressed bins first, and so is useful for examining what has been suppressed. Sorting by `Distortion` descending shows the bins with the most error first, and so is useful for examining which bins have high distortion.
 
+### Suppress bin
+
+The first row in the Anonymized Data table might contain the _suppress bin_, which provides the anonymized count of the data from the bins which got suppressed. The suppress bin has all of the columns selected for anonymization displaying the value `*`.
```

Comment:
Some ideas how to make this shorter (feel free to disagree, I find it difficult to find the right words):

The first row in the Anonymized Data table might contain the _suppress bin_, which provides the total anonymized count of all bins that were suppressed. The suppress bin shows all column values as `*`.

I'm between `total anonymized count` or `combined anonymized count`, or any other ideas?